### PR TITLE
Prac31

### PR DESCRIPTION
**Pull request by soniamitchell**

### CODE REVIEW

File: inst/tutorials/practical3-2/practical3-2.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Practical 3-2: Stochastic population growth model"
+title: "3-2: Stochastic population growth model"
```
Comment (richardreeve) [inline inst/tutorials/practical3-2/practical3-2.Rmd:2]: You seem to have reverted some of my changes? Was there a reason for that. All of the previous practicals have the word practical first...

File: inst/tutorials/practical3-1/practical3-1.Rmd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Practical 3-1: Writing an R package"
+title: "3-1: Writing an R package"
```
Comment (richardreeve) [inline inst/tutorials/practical3-1/practical3-1.Rmd:2]: Ditto here...

File: inst/dummy_package/0105-run-birth-death.R [hunk 3]
```diff
@@ -1,37 +0,0 @@
-#' # Simple birth-death difference equation model
```
Comment (richardreeve) [inline inst/dummy_package/0105-run-birth-death.R:1]: Why was this deleted? Don't the students need it for the second practical series?

**Review by richardreeve — COMMENTED** [review]
@soniamitchell I appreciate you've merged this, but I have a couple of questions about things I think need to be reverted... if so, I can do them in my current branch. I think maybe you had a merge conflict where you overrode my changes?

File: inst/dummy_package/0105-run-birth-death.R (hunk 3 above)
Comment (richardreeve) [reply inst/dummy_package/0105-run-birth-death.R:1]: Oh, sorry - this is in dummy_package - it should be deleted.

File: inst/tutorials/practical3-2/practical3-2.Rmd (hunk 1 above)
Comment (soniamitchell) [reply inst/tutorials/practical3-2/practical3-2.Rmd:2]: Sorry! I did have a merge conflict. But wasn’t given the option to resolve it. So I cloned the repo and added these files back in. Sorry! Feel free to change it back..

Comment (richardreeve) [reply inst/tutorials/practical3-2/practical3-2.Rmd:2]: Will do. No problem - I assumed something like that had happened.
